README.md

```diff
@@ -1,56 +1,181 @@
-# Hello-Project# HelloProject
+#Hello Microverse Project
 
-# Hello-Microverse# helloMicroverse
+# 📗 Table of Contents
 
-#Project Title
-Hello Microverse
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-#Project Description
-The Hello Microverse project is for testing the implementation of front end design and checking how does it work with linters
+# 📖 [Hello_microverse] <a href="README.md"> Hello_microverse</a>
 
-🛠 Built With
-Tech Stack
-vscode
-linters
+ <h1 class="header">Hello Microverse!</h1>
 
-#Describe the tech stack and include only the relevant sections that apply to your project.
-Key Features - using of linters to check the errors before reaching to backend
+> Describe your project in 1 or 2 sentences.
 
-🚀 Live Demo
-The project will be be deployed in Netlify
+**[hello_microverse]** is a simple project to set up a working linters configuration and well detailed file structure based on github workflows
 
-💻 Getting Started
-New developer could make use of your project by clone from my github account of "github/ghermaico135"
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>html</li>
+  </ul>
+</details>
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li>Css</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>none</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>None</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Zero html-css linters error]**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Not available at the moment]()
+
+## 💻 Getting Started
+
+> Describe how a new developer could make use of your project.
```

Comment:
- [ ] For the same case, kindly consider removing the default placeholders. This will improve your readme and make it more professional.